### PR TITLE
Blog post Test Automation with Limited Resources

### PR DESCRIPTION
**Pull request by derdualist**

### CODE REVIEW

File: source/posts/2024-02-26-test-automation-with-limited-resources.html.markdown.erb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+We encourage others to contribute to the libraries and use them to improve their own test automation processes.
+
+## Conclusion
+
+Test automation can be done, even with limited time and resources.
+In the end, it will probably pay off.
+If you find yourself in a similar situation, where you want to actively involve your existing QA
+in the test automation process, we recommend that you consider IBMiX's approach to test automation.  
+Take a look at the libraries and consider getting involved.  
+Leave a comment below to let us know what you think about this approach.
```
Comment (sriv) [inline source/posts/2024-02-26-test-automation-with-limited-resources.html.markdown.erb]: the gauge blog site does not have a commenting mechanism currently.

**Review by sriv — CHANGES_REQUESTED** [review]

**Comment by sriv** [comment]
Thank you for the blog @derdualist . The preview of your post lies here: https://deploy-preview-194--kind-hugle-dbff4b.netlify.app/2024/02/26/test-automation-with-limited-resources/

Please see if this matches your expectation.

**Comment by derdualist** [comment]
Everything seems fine now. Thanks for the review.

**Review by sriv — APPROVED** [review]

**Comment by sriv** [comment]
Thank you @derdualist . The post is live now - https://gauge.org/2024/02/26/test-automation-with-limited-resources/

**Comment by sriv** [comment]
My apologies on the commenting. I see that there is discus enabled. ( My laptop browser had a blocker which disabled it).

If you wish to add the last line back I will merge it.

Sorry again for the mixup.

**Comment by derdualist** [comment]
Ah ok, merging the last line back in would be nice. 
No worries. I appreciate we could leave a posting, thanks a lot!

**Pull request merged**